api/model/src/main/java/org/projectnessie/model/IcebergView.java

```diff
@@ -49,11 +49,10 @@ public abstract class IcebergView extends IcebergContent {
 
   public abstract int getSchemaId();
 
-  @NotBlank
-  @jakarta.validation.constraints.NotBlank
-  @NotNull
-  @jakarta.validation.constraints.NotNull
-  public abstract String getSqlText();
+  @Value.Default
+  String getSqlText() {
+    return "dummy sql text";
```

Comment:
`dummy sql text` does not look nice in an API :) why not `null` now that this field is nullable?

Reply:
renamed to "default sql text". Suggestions are welcome.